scripts/install_software.sh

```diff
@@ -14,10 +14,12 @@ if [[ $# -eq 3 ]]; then
    YB_DOWNLOAD_LOCATION=$3
```

Comment:
I don't think we should have editions anymore.. What if the user wants to deploy a different version other than 1.3.0.0. In the current approach that would not work. 

Also why are we using standalone scripts, we should use the common utilities repo.

Reply:
Cloudformation script is different than the utility repo script that's why we are using the standalone scripts as of now. In a future release, we will try to use common utility repo.